docs/reference/upgrade/rolling_upgrade.asciidoc

```diff
@@ -7,11 +7,27 @@ a time so upgrading does not interrupt service. Running multiple versions of
 not supported, as shards cannot be replicated from upgraded nodes to nodes
 running the older version.
 
-It is best to upgrade the master-eligible nodes in your cluster after all of
-the other nodes. Once you have started to upgrade the master-eligible nodes
-they may form a cluster that nodes of older versions cannot join. If you
-upgrade the master-eligible nodes last then all the other nodes will not be
-running an older version and so they will be able to join the cluster.
+We strongly recommend you upgrade you divide your cluster's nodes into the
```

Comment:
"We strongly recommend you upgrade you divide your cluster's nodes" -> "We strongly recommend that when you upgrade you divide your cluster's nodes"

Reply:
Good catch, thanks